.simpleUpdater/testSimpleUpdater-win32-x64.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+original_version=$(node ./.simpleUpdater/updateVersion.js 0.0.1)
+
+npm install
+npm run make
+node ./.simpleUpdater/updateVersion.js $original_version
+
+sed -i -e '$a\' package.json
+npm install
+
+./out/make/squirrel.windows/x64/cucumber-forge-desktop-win32-x64-0.0.1-Setup.exe
+
+echo "
+To validate that the auto-update functionality works, the current Cucumber Forge code
+has been packaged with an old version and installed. Restart the application to 
+trigger the automatic check for updates. Make sure that the updator properly downloads 
+and installs the latest version."
```

Comment:
have you tested this on any machine but yours with sh installed?

Reply:
#WorksOnMyMachine  No, I have not.  Is there something you are concerned about here?  Feel free to pull this down and try it on yours....

Reply:
All of mine also have bash/sh set up, so I don't think that'd match the point. I'm just worried that when someone tries this bash script on a standard windows machine that it won't work, as it doesn't have bash.

Reply:
Just might need a note for 1 that you'll need bash functional on your machine.

Reply:
Fair point.  I will add a note to the RELEASE.md that you will need to be on Windows and running a bash shell (such as Git Bash for Windows).